ceph-pr-render-docs/build/build

```diff
@@ -15,6 +15,8 @@ if [ -z "$ghprbPullId" ]; then
     PR_ID="manual"
 fi
 
+# Cleanup docs rendered 90+ days ago
+find /var/ceph-prs/ -mindepth 1 -maxdepth 1 -mtime +90 -exec rm -rvf {} \;
```

Comment:
if for whatever reason this fails, the `set -ex` at the beginning of this file will cause the job to fail before building the docs. Can this unset the `e` flag? (I think this is done with `+e`) or placed at the very end?